src/event-handler/index.ts

```diff
@@ -0,0 +1,85 @@
+import { EventEmitter } from 'events';
+import Ain from '../ain';
+import {
+  BlockchainEventTypes,
+  BlockFinalizedEventConfig,
+  EventConfigType,
+  ValueChangedEventConfig,
+} from '../types';
+import EventFilter from './event-filter';
+import EventChannelManager from './event-channel-manager';
+
+export default class EventHandler {
```

Comment:
As 'EventHandler' is already used on the server side (i.e., blockchain node), what about using a different name e.g. 'EventManager'?

Reply:
Looks good. Done